Contract7/Hendrix/main.py

```diff
@@ -0,0 +1,104 @@
+import pygame, sys, colorsys
+from pygame.locals import *
+from PIL import Image
+
+pygame.init()
+
+#defining colors
+PURPLE = 191, 55, 131
+YELLOW = 250, 212, 27
+WHITE = 255, 255, 255
+BLACK = 0, 0, 0
+
+# screen dimentions
+WIDTH = 400
+HEIGHT = 600
+
+angle = -90  # degrees turned by corner spirals
+timer = 0  # initialising timer variable for while loop
+clock = pygame.time.Clock()
+
+# creating mask to go over side dots
+maskHeight = 15
+dotBlock = pygame.Surface((15, maskHeight))
+dotBlock.set_alpha(90)
+dotBlock.fill(BLACK)
+maskXleft = 5
+maskXright = 380
+maskY = 504
+blockDist = 0
+maskUp = True
+
+# mask for top dots
+
+# creating screen
+screen = pygame.display.set_mode((WIDTH, HEIGHT), 0, 32)
+
+def loadscale(image, width, height):
+    return pygame.transform.scale(pygame.image.load(image),(width, height))
+
+# gets background image and creates another with purple and yellow inverted
+bgImage = loadscale("Images/BaseImage.png", WIDTH, HEIGHT)
+
+# Inverted purple and yellow background
+bgImage2 = loadscale("Images/BaseImage.png", WIDTH, HEIGHT)
+bgImageArray = pygame.PixelArray(bgImage2)
+pygame.PixelArray.replace(bgImageArray, PURPLE, WHITE, 0)  # changes purple to white temporarily

```

Comment:
Could be changed/ removed to be more efficient rather then having temporary colour states.